.github/workflows/build-crates-individually.patch.yml

```diff
@@ -25,13 +25,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3.5.2
 
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
-          override: true
+      # Setup Rust with stable toolchain and minimal profile
+      - name: Setup Rust
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=stable --profile=minimal
```

Comment:
What is the security impact of this change to using a bare URL? Did the action used to verify the downloaded binaries?

Reply:
What's the performance impact of this change?

Reply:
> What is the security impact of this change to using a bare URL? Did the action used to verify the downloaded binaries?

It looks like this is exactly what the maintained action does:
https://github.com/dtolnay/rust-toolchain/blob/b44cb146d03e8d870c57ab64b80f04586349ca5d/action.yml#L65

That action also applies some additional environmental variables, I've added them to the Zebra repository settings:
- CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
- CARGO_TERM_COLOR=always

This should speed up builds and show output nicely.